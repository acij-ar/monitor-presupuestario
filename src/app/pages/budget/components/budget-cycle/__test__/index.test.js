const React = require('react');
const { shallow } = require('enzyme');

const BudgetCycle = require('..');

describe('BudgetCycle component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<BudgetCycle/>);
    expect(wrapper).toMatchSnapshot();
  });
});
