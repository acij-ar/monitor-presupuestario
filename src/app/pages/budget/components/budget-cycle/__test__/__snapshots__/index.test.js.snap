// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BudgetCycle component should match snapshot 1`] = `
<ContentSection
  id="budget-cycle"
  title="El ciclo del presupuesto"
>
  <p>
    El presupuesto del Estado pasa por distintas etapas, antes, durante y después de lo que se suele denominar “ejercicio fiscal”. Este término alude al periodo de 12 meses en los que se ejecuta un determinado presupuesto.
  </p>
  <p>
    Así, como veremos a continuación, el presupuesto del año 2019 se elaboró, discutió y sancionó en 2018, y será evaluado en 2020.
  </p>
  <p>
    En cada una de esas etapas, diferentes organismos generan distintos documentos en los que se encontrará buena parte de la información necesaria para nuestro análisis. A continuación, vamos a explicar de manera sencilla cómo leerlos y qué información puede extraerse de cada uno. Las distintas fases de un ciclo presupuestario completo pueden resumirse del siguiente modo:
  </p>
  <ol>
    <li>
      Formulación
    </li>
    <li>
      Discusión y aprobación
    </li>
    <li>
      Ejecución
    </li>
    <li>
      Control y rendición de cuentas
    </li>
  </ol>
</ContentSection>
`;
